cmd/kogito/command/deploy/deploy_service_test.go

```diff
@@ -331,3 +331,71 @@ func Test_WithManagementConsole(t *testing.T) {
 	assert.Contains(t, lines, "successfully created")
 	assert.Contains(t, lines, mgmtConsoleURI)
 }
+
+func Test_DeployCmd_WithCustomArtifactDetails(t *testing.T) {
+	ns := t.Name()
+	groupId := "com.mycompany"
+	artifactId := "testproject"
+	version := "2.0-SNAPSHOT"
+	fileLocation, _ := url.Parse("https://raw.githubusercontent.com/kiegroup/kogito-examples/master/dmn-quarkus-example/src/main/resources/Traffic Violation.dmn")
+	cli := fmt.Sprintf("deploy-service example-from-file %s -p %s --project-group-id %s --project-artifact-id %s --project-version %s", fileLocation, ns, groupId, artifactId, version)
+
+	ctx := test.SetupCliTest(cli,
+		context.CommandFactory{BuildCommands: BuildCommands},
+		&corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: ns}},
+		&apiextensionsv1beta1.CustomResourceDefinition{ObjectMeta: metav1.ObjectMeta{Name: v1alpha1.KogitoAppCRDName}})
+	// Start the test
+	lines, _, err := test.ExecuteCli()
+	assert.NoError(t, err)
+	assert.Contains(t, lines, "successfully created")
+
+	// This should be created, given the command above
+	kogitoApp := &v1alpha1.KogitoApp{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "example-drools",
+			Namespace: ns,
+		},
+	}
+
+	exist, err := kubernetes.ResourceC(ctx.Client).Fetch(kogitoApp)
+	assert.NoError(t, err)
+	assert.True(t, exist)
+	assert.NotNil(t, kogitoApp)
+	assert.Equal(t, groupId, kogitoApp.Spec.Build.Artifact.GroupId)
+	assert.Equal(t, artifactId, kogitoApp.Spec.Build.Artifact.ArtifactId)
+	assert.Equal(t, version, kogitoApp.Spec.Build.Artifact.Version)
+}
+
+func Test_DeployCmd_WithDefaultArtifactDetails(t *testing.T) {
+	ns := t.Name()
+	groupId := "com.company"
+	artifactId := "project"
+	version := "1.0-SNAPSHOT"
+	fileLocation, _ := url.Parse("https:ss//raw.githubusercontent.com/kiegroup/kogito-examples/master/dmn-quarkus-example/src/main/resources/Traffic Violation.dmnsd")
```

Comment:
Same here regarding the `testdata` directory.

Reply:
edit test to remove remote file location